airflow/www/security.py

```diff
@@ -27,7 +27,7 @@
 from airflow.exceptions import AirflowException
 from airflow.utils.db import provide_session
 from airflow.utils.log.logging_mixin import LoggingMixin
-from airflow.www.app import appbuilder
+from airflow.www.app import app, appbuilder
```

Comment:
I had initially used appbuilder, like was done in `get_user_roles` but that caused all of the tests to fail, so I started directly using `app.config`. Is there something I might have been missing?